WcaOnRails/config/locales/en.yml

```diff
@@ -806,7 +806,7 @@ en:
         title: "What are the WCA accounts for? What is the difference between WCA accounts and WCA profiles?"
         #context: Paragraph of the answer to "What are the WCA accounts for? ..."
         content:
-          1: "You are assigned a WCA ID when you first participate in a WCA sanctioned competition. When the results from that competition are uploaded, your WCA profile will be created, with results from every competition you have been to."
+          1: "You are assigned a WCA ID when the results from your first WCA sanctioned competition are uploaded."
```

Comment:
Maybe a parenthetical about how this could be up to a week after the competition is over?